api/teams.go

```diff
@@ -22,6 +22,30 @@ type TeamResult struct {
 	Body    *primitive.Team `json:"body"`
 }
 
+// List retrieves all teams for an org based on the filtered values
+func (t *TeamsClient) List(ctx context.Context, orgID *identity.ID, name, teamType string) ([]TeamResult, error) {
+	v := &url.Values{}
+
+	if orgID != nil {
```

Comment:
the end-point doesn't support it today, so I'd rather match what it supports.

Over time, I think these endpoints will fade away as we force more things into the larger data structures.